apps/embed/templates/meinberlin_embed/embed.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+{% load static %}
+<html class="embed">
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Embedded meinBerlin</title>
+    <link rel="shortcut icon" type="image/x-icon" href="/favicon.ico" />
+    <link rel="stylesheet" type="text/css" href="{% static 'vendor.css' %}" />
+    <link rel="stylesheet" type="text/css" href="{% static 'meinberlin.css' %}" />
+    <link rel="stylesheet" type="text/css" href="{% static 'embed.css' %}" />
+    <script src="{% url 'javascript-catalog' %}"></script>
+    <script src="{% static 'vendor.js' %}"></script>
+    <script src="{% static 'meinberlin.js' %}"></script>
+    <script src="{% static 'embed.js' %}"></script>
+    <meta name="viewport" content="width=device-width" />
+    {% block header %}{% endblock %}
+</head>
+
+<body>
+    <header class="embed-header">
+        <a>Login</a>
+    </header>
+    <main class="embed-main">
+        Embed
```

Comment:
I'd prefer to have this textnode removed, it flashes otherwise when the content got loaded.